### PR TITLE
Add a form to create a PoliticalEntity

### LINKED ISSUE #14: Add form for Nation creation/edition

### PR DESCRIPTION
**Pull request by amaury1093**
- ~Not quite finished yet, still has `wikipedia_link` instead of `links`, waiting for https://github.com/chronoscio/backend/pull/30~ Added `links` field.
- Not wired up to the backend. On submit, check console to see the result.
- Cannot input Diplomatic Relation yet, left for another PR.

Closes #14.

### CODE REVIEW

**Comment by quorth0n** [comment]
Will #19 be addressed in a separate PR?

File: packages/app/components/Nation/NewNation/NewNation.tsx
Comment (quorth0n) [inline packages/app/components/Nation/NewNation/NewNation.tsx:35]: Is there any plan to further validate the references textarea to be wikipedia-style or do you want to just leave this up to the backend for v1?

**Review by quorth0n — COMMENTED** [review]
image
Is this input not valid? The submit button is disabled.

File: packages/app/components/Nation/NewNation/NewNation.tsx
Comment (amaury1093) [reply packages/app/components/Nation/NewNation/NewNation.tsx:35]: Yes, didn't add it on purpose because I wanted to make a decision on https://github.com/chronoscio/backend/issues/28 first.

If we go the regex way, I'll add it to the frontend too.
If we go the new Reference model way, we define fields in the new model and I'll add validation on those fields.

**Comment by amaury1093** [comment]
I'll address #19 in a separate PR, this one is already big enough.

On your screenshot, my guess is that the reference is more than 150 chars. But the fact that it doesn't show an error popup is a bug, I'll fix that in this PR.

**Comment by amaury1093** [comment]
In the latest 3 commits, I created an `ui` package, where I put all common UI elements used across the app. Might be easier to browse https://github.com/chronoscio/frontend/tree/am-input/packages instead of looking at file diffs. There are no code changes in the last 3 commits.

**Review by quorth0n — CHANGES_REQUESTED** [review]
I've been hitting the same bug I posted earlier with valid data, the submit button is disabled, however I managed to pinpoint it this time. It will not let you submit until the disputed territory switch has been toggled or otherwise changed. You can flick it back to complete control and it will work, but until it's changed the submit button stays disabled.

Code-wise everything LGTM. I'll merge after this is fixed.

**Review by quorth0n — APPROVED** [review]

**Comment by quorth0n** [comment]
^approving in advance as I suspect this will be an easy/quick fix.

**Comment by amaury1093** [comment]
@whirish Ah ok, didn't have this problem on my opera or firefox. Should be fixed now, I'll let you merge.

**Review by quorth0n — APPROVED** [review]

**Pull request merged**
